tests/gpflow/test_monitor.py

```diff
@@ -21,6 +22,14 @@ class Data:
     num_steps = 2
 
 
+class DummyMonitor(MonitorTask):
+    def run(self, **kwargs):
+        pass
+
+    def get_internal_step(self):
+        return self.current_step
```

Comment:
```suggestion
class DummyTask(MonitorTask):
    def run(self, **kwargs):
        pass
```
This name makes a bit clearer what we're doing. Also, in the test we can access `current_step` directly, no need for the method indirection.